### PR TITLE
Use prettier to format .md files

### PR DESCRIPTION
**Pull request by webmaster128**
First step for #829

The idea is to format code and text separatly
- since different width is desired,
- to avoid unnecessary build times during development.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
   "scripts": {
     "bootstrap": "lerna bootstrap",
     "format": "lerna run format",
```
Comment (willclarktech) [inline package.json]: I would expect the `format` script to cover the root as well.

File: package.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
...
+    "format-text": "prettier --write --prose-wrap always --print-width 80 \"./*.md\" \"./docs/**/*.md\" && lerna run format-text",
```
Comment (willclarktech) [inline package.json]: This recurses, no?

File: package.json (hunk 1 above)
Comment (webmaster128) [reply package.json]: The default "format" implementation is `prettier --write --loglevel warn \"./src/**/*.ts\"`, which does not do anything in the root dir. Thus I did not add it here.

File: package.json (hunk 2 above)
Comment (webmaster128) [reply package.json]: No:

```
lerna run <script>

Run an npm script in each package that contains that script
```

i.e. `lerna run format-text` runs `yarn format-text` for each folder in `packages/` only.

Comment (willclarktech) [reply package.json]: Ah, right. `lerna run` not `yarn run`.

**Comment by webmaster128** [comment]
> There are a few situations where prettier requires more than one pass to reach a stable point, which is possibly what's happened here.

In this case the change from

```
------------------ Rough ideas not quite ready for backlog ------------------------
```

to 

```
------------------ Rough ideas not quite ready for backlog
------------------------
```

caused the second run to interprete it as a h2. Will fix manually

**Pull request merged**